CONTRIBUTING.md

```diff
@@ -66,11 +66,19 @@ compromise among collaborators be the default resolution mechanism.
 
 # Becoming a Member and Collaborator
 
-To become a member of the Community Committee, start participating! Attend the bi-weekly meetings, investigate issues tagged as `good first issue`,
+To become a Member of the Community Committee, start participating! Attend scheduled meetings, investigate issues tagged as `good first issue`,
 file issues and pull requests, and provide insight via GitHub. Request to become an Observer by
 filing an issue. Once added as an Observer to meetings, we will track attendance and participation
-for 3 months, as according to our [governance guidelines](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md#section-4-establishment-of-the-community-committee). If you meet the minimum
-attendance and are participating, the CommComm will vote to add you as a member.
+according to our [governance guidelines](https://github.com/nodejs/community-committee/blob/master/GOVERNANCE.md#section-4-establishment-of-the-community-committee).
+Once you meet the minimum attendance and participation requirements of a Member, you will automatically become a Member.
```

Comment:
I am -1 on this change. Automatic membership because someone shows up may not necessarily be helpful. I can very easily see someone _just_ attending meetings and not contributing to the CommComm in other meaningful ways to become a voting member. This, in effect, recreates some of the issues that we've had with the Individual Membership representatives in the past.

Additionally, if membership becomes automatic I would be a strong proponent of more active and strict removal processes for members to ensure that we are engaged with the Committee rather than absentee.

Reply:
So, I'll say something that I'm sure lots of @nodejs/tsc and @nodejs/community-committee members could chime in on through past discussions–what could we use as a measure of 'showing up and doing'? The 'showing up' we have documented as meeting attendance and participation in discussions. Instead of voting on people to be brought onboard(which I think can instill whispers of discourse if it doesn't go well and we want everyone to feel welcome so long as they are following the CoC + working towards the common values), what if we make the other requirement commits and/or pull requests? It doesn't need to be commits merged, per se, but that the observer working towards member is making an active effort to contribute?

Reply:
I am a huge +1 to that, @hackygolucky - we're at the point where meeting attendance isn't the only path to contributing to the CommComm, and are actively looking to grow and diversify the work we're doing.

I've known of several individuals who just can't make a meeting because of time zones or personal blockers, and as such lose out on that 1/6th of their marks of attendance toward membership. 

We need to address this at a fundamental level to enable more people to participate in ways that are positive and helping the CommComm grow.

@williamkapke could you advise on what we'd need to do/what changes would need to be made for us to approach this?

Reply:
Yeah- I wasn't under the impression that `attendance` === `min requirements`. But, do recall that we don't have an answer to what "participation" can be defined as.

I support working towards finding a definition. If anyone has a strawman idea to throw out- please open a PR and let's see if we can hash it out.

I'd suggest that we do it in the Contribution policy.

Reply:
@williamkapke I just tried to start in that, but the ["Becoming a Member"](https://github.com/nodejs/community-committee/blob/master/CONTRIBUTING.md#becoming-a-member-and-collaborator) section actively references the Governance as a source of truth. Is that a blocker for hashing it out in CONTRIBUTING.md?

Reply:
governance.md states that `CC members are expected to regularly participate in CC activities.`. i think we can use that as a baseline because it's a very vague definition